scaleout/stackn/templates/event-listener-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: studio-event-listener
+  name: studio-event-listener
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      name: studio-event-listener
+  template:
+    metadata:
+      labels:
+        name: studio-event-listener
+        app: stackn-studio
+        allow-api-access: "true"
+    spec:
+      automountServiceAccountToken: true
+      serviceAccountName: studio
+      containers:
```

Comment:
lack of security context, see studio and other deplyments, requires that that image is run as non-root